src/transformers/pipelines/text_to_audio.py

```diff
@@ -43,6 +43,29 @@ class TextToAudioPipeline(Pipeline):
 
     Learn more about the basics of using a pipeline in the [pipeline tutorial](../pipeline_tutorial)
 
+    <Tip>
+
+    You can specify parameters passed to the model generation/forward method by using
+    [`TextToAudioPipeline.__call__.forward_params`].
+
+    Example:
+
+    ```python
+    >>> from transformers import pipeline
+
+    >>> music_generator = pipeline(task="text-to-audio", model="facebook/musicgen-small", framework="pt")
+
+    >>> # diversify the music generation by adding randomness with a high temperature and set a maximum music length
+    >>> forward_params = {
+    ...     "do_sample": True,
+    ...     "temperature": 0.7,
+    ...     "max_new_tokens": 35,
+    ... }
+
+    >>> outputs = music_generator("This is a test", forward_params=forward_params)
```

Comment:
Do you think it would help for consistency allowing the pipeline to accept either `forward_params` or `generate_kwargs`? If `forward_params`, then it gets passed to `.forward` or `.generate` (whichever one the model does). If `generate_kwargs`, it only gets passed if the model generates with `.generate`. IMO this would help with consistency across pipelines

Reply:
I suppose this would make sense in terms of consistency, but not in terms of simplicity, as it might be confusing for users to have two dictionaries doing the same thing. Anyway, I'm happy with your solution, since users can still refer to the documentation in case of confusion.

WDYT @ArthurZucker and @Narsil ? 

I'll open a PR once we agree on that!

Reply:
more aligned with sanchit here, generation params should be passed to `generate_kwargs` for sure and not as `forward_params` it's confusing and not consistent with our API